docs/integrations/amazon-aws/ses.md

```diff
@@ -289,7 +289,7 @@ See the overview of SES CloudTrail events including failed, and successful event
 
 See the details of various SES CloudTrail events including the identity, get send, domain, receipt, and email address events.
 
-<img src={useBaseUrl('img/integrations/amazon-aws/SES-Notification.png')} alt="AWS API Gateway" />
+<img src={useBaseUrl('img/integrations/amazon-aws/AmazonSESCloudTrailEventsbyEventName.png')} alt="AWS SES Cloud Trail Events by Event Name" />
```

Comment:
It is broken when previewed, kindly check.

Reply:
https://github.com/SumoLogic/sumologic-documentation/pull/2593/files#diff-7c72755f83e5f7bd574a60da82c3abdb4b72b534bd3763280b18eea09aadb2d2

Already added in the PR.

Reply:
Please check once